general/src/protocol/contexts.rs

```diff
@@ -264,8 +264,6 @@ pub enum Context {
     Trace(Box<TraceContext>),
     /// Information related to Monitors feature.
     Monitor(Object<Value>),
```

Comment:
This can probably also go then.

Reply:
Monitor should stay in the format for now, we should just extend it eventually.